### PR TITLE
working calendar as basic navigation, no SSR for day view so far

### PR DESCRIPTION
**Pull request by martin-havala**
add basic calendar component with possibility to list days [range so far based on basic min-max selection form JournalEntries ], no big styling for now

added laravel debugbar
added dayjs (light momentjs replacement)

### CODE REVIEW

File: resources/js/vendor.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    window.Popper = require('popper.js').default;
+    window.$ = window.jQuery = require('jquery');
+
+    require('bootstrap');
+} catch (e) {}
+
+
+window.Vue = require('vue');
+window.axios = require('axios');
+window.axios.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
```
Comment (eronisko) [inline resources/js/vendor.js]: How about moving thisinto app.js? IMHO there's not much value to keep this bootstrapping code in two separate files.

File: resources/views/day.blade.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        data: {
+            daysAvailable: <?php print($daysAvailable ?? '') ?>,
+            startAt: '<?php print($date ?? '') ?>',
+            content: <?php print($entries ?? '') ?>,
+        },
+        methods: {
+            // wannabe vue-router replacement for now
+            dayClbck: function($day) {
+                this.startAt = $day;
+                axios.get("../api/day/" + $day).then((res) => {
```
Comment (eronisko) [inline resources/views/day.blade.php]: I'd use a URL helper here too: https://laravel.com/docs/6.x/helpers#method-route

File: routes/web.php [hunk 3]
```diff
@@ -14,3 +14,40 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+
+Route::get('/day', function () {
```
Comment (eronisko) [inline routes/web.php]: I'd suggest moving this into a controller.

File: resources/views/day.blade.php [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                </div>
+            </transition-group>
+        </div>
+    </div>
+</body>
+<script defer>
+    const calendar = new Vue({
+        el: '#calendar',
+        data: {
+            daysAvailable: <?php print($daysAvailable ?? '') ?>,
```
[the remark this answers is not in the document]
Comment (martin-havala) [reply resources/views/day.blade.php]: it does, but auto-formatter makes it ugly and useless

[the remark this answers is not in the document]
Comment (eronisko) [reply resources/views/day.blade.php]: Autoformatter? Do you mean your editor/IDE?

File: resources/js/vendor.js (hunk 1 above)
Comment (igor-kamil) [reply resources/js/vendor.js]: isn't it better to generate `vendor.js` just by calling `.extract()` method in `/webpack.mix.js` ?

**Pull request merged**